geoengine_base_geolocalize/views/res_partner_view.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record model="ir.ui.view" id="geoengine_partner.geo_partner_view_form">
```

Comment:
`geoengine_partner` in the `id` looks odd :confused: 
IMO would be better to use it in `inherit_id` if the goal is to change the location of the field in the view...